octopus_deploy/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,283 @@
+name: Octopus Deploy
+files:
+- name: octopus_deploy.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+    - template: init_config/http
+  - template: instances
+    options:
+    - name: octopus_endpoint
+      display_priority: 8
+      description: |
+        Octopus API endpoint.
+        See https://octopus.com/docs/octopus-rest-api/getting-started#authentication for more details.
+      value:
+        example: http://localhost:80
+        type: string
+      required: true
+    - name: spaces
+      display_priority: 5
+      description: |
+        Optional configuration to indicate the spaces that we want to be processed. If not configured,
+        all spaces and their corresponding project groups and projects will be processed.
+
+        The 'include' key will indicate the regular expressions of the spaces for which metrics are to be reported
+        and the configuration to be applied to each of them. Each group may have a 'project-groups'-like configuration,
+        enabling or disabling metric collection for the project groups which match that condition. For further details
+        see next section 'project-groups'. If no configuration associated with the key is indicated with the regular
+        expression, they will be processed with the default configuration.
+
+        The spaces will be processed in the order indicated in the 'include'.
+        If a space is matched on an 'include' key, it will only be processed there and not in a later 'include'
+        that it might match on.
+
+        The 'exclude' key will indicate the regular expressions of those spaces for which metrics
+        are not to be reported.
+        The excludes will have priority over the includes, that is, if a space matches an exclude, it will not be
+        processed even if it matches an include. The 'include' key must be used if using the 'exclude' key.
```

Comment:
```suggestion
        The 'exclude' key indicates the regular expressions of spaces where metrics
        are not reported.
        The excludes have priority over the includes; that is, if a space matches an exclude, it will not be
        processed even if it matches an include. The 'include' key must be used when using the 'exclude' key.
```